docs/dev/notebooks.md

```diff
@@ -0,0 +1,66 @@
+# Notebooks guidelines 
+
+Our guides and tutorials are frequently written using iPython Notebooks. The notebooks require specific formatting, are continuosly tested (when possible) and we have a specific process to manage the lifecycle of a notebook before and after a Cirq release.    
+
+## Formatting 
+
+Formatting is easy, the script `check/nbformat` should tell you if your notebooks are formatted or not.
+You can apply the changes in one go with `check/nbformat --apply`. It is recommended to add this to your [git pre-commit hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks), to save feedback time and CI resources. 
+
+## Output cells 
+
+Output cells typically should not be saved in the notebook. They will be generated for the final site.
+The exception to this rule are notebooks with external dependencies ([see below](#notebooks-with-external-dependencies)). 
+
+## Header
+
+We also expect a standard header to be included in all of our notebooks: 
+- the links to colab, github and the main site ([quantumai.google/cirq](https://quantumai.google/cirq))
+- optional package installation (you can assume Colab dependencies exist)
+
+
+Example header:
+
+![notebook header](../images/notebook_header.png)
```

Comment:
I think this image is not uploaded in this PR?

Reply:
hehe. uploaded.